python/ray/autoscaler/v2/tests/util.py

```diff
@@ -15,6 +15,13 @@ def make_autoscaler_instance(
     ray_node: Optional[autoscaler_pb2.NodeState] = None,
     cloud_instance_id: Optional[str] = None,
 ) -> AutoscalerInstance:
+
+    if cloud_instance_id:
```

Comment:
Could we initialize the parameters in the AutoscalerInstance constructor? If we can initialize it in the constructor, we can remove this function. This function is pretty shallow.